php/class-post-type.php

```diff
@@ -1082,4 +1127,69 @@ public function show_publish_error_admin_notice() {
 		</div>
 		<?php
 	}
+
+	/**
+	 * Get conflicts settings
+	 *
+	 * @param \WP_Post $post post to compare conflict values.
+	 * @param array    $settings setting to search for optional.
+	 *
+	 * @return array
+	 */
+	public function get_conflicted_settings( $post, $settings = array() ) {
+		global $wpdb;
+		if ( $post && static::SLUG === get_post_type( $post ) ) {
+			$post = get_post( $post );
+		}
+		$conflicted_settings = array();
+		if ( empty( $settings ) ) {
+			$content = $this->get_post_content( $post );
+			if ( empty( $content ) || ! is_array( $content ) ) {
+				return $conflicted_settings;
+			}
+			$settings = array_keys( $content );
+			if ( empty( $settings ) ) {
+				return $conflicted_settings;
+			}
+		}
+		$query = $wpdb->prepare( "SELECT ID, post_name, post_title, post_status, post_content FROM $wpdb->posts WHERE post_type = %s AND post_status IN ( 'pending', 'future' ) ", static::SLUG );
+		// Todo: finalize post_status to check in.
```

Comment:
Is this todo done?

Reply:
Do we want to keep draft or pending and future are fine? This will filter post which we will be searching conflicted settings in. @westonruter

Reply:
Limiting to `pending` and `future` make sense to me. I could also see `draft` being included.  Maybe include `draft` for now and then we can remove it if it gets too noisey?